src/hosted-fields/external/hosted-fields.js

```diff
@@ -436,8 +436,14 @@ function HostedFields(options) {
     }
 
     field = options.fields[key];
+    if(typeof field.selector == 'string') {
```

Comment:
This doesn't fit our code style (our fault for not having travis set up)

You can run `npm run lint` to run the eslint command.

I've just merged a PR to add a travis ci integration, so if you merge in master, your PR will run a build.

Reply:
Yep, I really only created this PR to illustrate, as an example what the issue is, and a quick 2 sec fix. Probably not the best solution. I'm buried in client work at the moment and don't have time to get the whole dev environment set up, write tests, docs, review code style policies, etc... so go ahead and close out the PR - it was just meant as an enhancement to the issue. Also, agree with the api change, container makes more sense. Thanks!

Reply:
OK, I'll close this PR, but leave the issue open for our backlog.